tests/foreman/cli/test_variables.py

```diff
@@ -752,235 +1145,139 @@ def test_negative_create_matcher_avoid_duplicate(self):
+            smart_variable['override-values']['avoid-duplicates'], True)
 
     @run_only_on('sat')
-    @stubbed()
-    @tier3
+    @tier1
     def test_negative_enable_avoid_duplicates_flag(self):
-        """Disable Avoid duplicates flag for non supported types.
+        """Attempt to enable Avoid duplicates flag for non supported types.
 
         @id: ea1e8e31-4374-4172-82fd-538d29d70c03
 
         @steps:
 
-        1.  Set variable type other than array.
+        1.  Create smart variable with type that is not array/hash.
+        2.  Attempt to update smart variable and set corresponding flags to
+        True state.
 
         @assert:
 
         1.  The '--merge-overrides' is only allowed to set to true for type
         hash.
         2.  The '--avoid-duplicates' is not allowed to set to true for type
         other than array.
-
-        @caseautomation: notautomated
-
-        @CaseLevel: System
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier3
-    def test_positive_impact_delete_attribute(self):
-        """Impact on variable after deleting associated attribute.
-
-        @id: ac6f3a65-ed39-4e97-bdee-349f08bd878e
-
-        @steps:
-
-        1.  Create a variable with matcher for some attribute.
-        2.  Delete the attribute.
-        3.  Recreate the attribute with same name as earlier.
-
-        @assert:
-
-        1.  The matcher for deleted attribute removed from variable.
-        2.  On recreating attribute, the matcher should not reappear in
-        variable.
-
-        @caseautomation: notautomated
-
-        @CaseLevel: System
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier1
-    def test_positive_create_override_from_attribute(self):
-        """Impact on variable on overriding the variable value from attribute.
-
-        @id: 07622cab-26f9-4826-a61b-64748e4866a1
-
-        @steps:
-
-        1.  Create a variable.
-        2.  Associate variable with fqdn/hostgroup.
-        3.  From host/hostgroup, override the variable value.
-
-        @assert:
-
-        1.  The host/hostgroup is saved with changes.
-        2.  New matcher for fqdn/hostgroup created inside variable.
-
-        @caseautomation: notautomated
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier1
-    def test_negative_create_override_from_attribute(self):
-        """No impact on variable on overriding the variable
-        with invalid value from attribute.
-
-        @id: 471fd4b3-c88a-4d3d-a142-343c75803c36
-
-        @steps:
-
-        1.  Create a variable.
-        2.  Associate variable with fqdn/hostgroup.
-        3.  From host/hostgroup, Attempt to override the variable with invalid
-        key type of value.
-
-        @assert:
-
-        1.  No matcher for fqdn/hostgroup is created inside variable.
-
-        @caseautomation: notautomated
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier1
-    def test_negative_create_empty_value_matcher_from_attribute(self):
```

Comment:
There is no way to interact with smart variables from host interface like we have in UI, so it is invalid case